chaos_genius/utils/metadata_api_config.py

```diff
@@ -5,6 +5,7 @@
     "BigQuery": False,
     "Snowflake": True,
     "Druid": False,
+    "AWS Athena": True
```

Comment:
This is going to throw error
```suggestion
    "AWS Athena": True,
```